src/EnergyPlus/SimulationManager.cc

```diff
@@ -499,6 +499,8 @@ namespace SimulationManager {
 				isFinalYear = true;
 			}
 
+			HVACManager::ResetNodeData(); // Reset here, because some zone calcs rely on node data (e.g. ZoneITEquip)
+
```

Comment:
OK, ResetNodeData is called, literally, at the beginning of the next environment.